### PR TITLE
Fix e2e-test framework

### PR DESCRIPTION
**Pull request by Kidswiss**
## Summary

* Some fixes to make the e2e-test work

## Checklist

- [x] Categorize the PR by setting a good title and adding one of the labels:
      `bug`, `enhancement`, `documentation`, `change`, `breaking`, `dependency`
      as they show up in the changelog
- [x] Update tests.
- [x] Link this PR to related issues.

### CODE REVIEW

File: package/crossplane.yaml [hunk 1]
```diff
@@ -17,7 +17,7 @@ metadata:
       [vshn/provider-minio](https://github.com/vshn/provider-minio/issues) repo.
 spec:
   controller:
-    image: ghcr.io/vshn/provider-minio:latest
+    image: ghcr.io/vshn/provider-minio/controller:latest
```
Comment (zugao) [inline package/crossplane.yaml:20]: Can't we have only one image?

**Review by zugao — COMMENTED** [review]
One thing

File: package/crossplane.yaml (hunk 1 above)
Comment (Kidswiss) [reply package/crossplane.yaml:20]: AFAIK not, one "image" is the package and the other the actual controller.

It's the same way for cloudscale and exoscale.

**Review by zugao — APPROVED** [review]
LGTM

**Pull request merged**